dartsim/src/WorldFeatures_TEST.cc

```diff
@@ -190,6 +196,12 @@ TEST_F(WorldFeaturesFixture, Gravity)
     EXPECT_PRED_FORMAT2(vectorPredicate3,
                         Eigen::Vector3d(0.5, 0, 2.5),
                         pos);
+    // pose for link without gravity should not change
+    Eigen::Vector3d posNoGravity = linkNoGravity->FrameDataRelativeToWorld()
+                        .pose.translation();
+    EXPECT_PRED_FORMAT2(vectorPredicate3,
+                        Eigen::Vector3d(10, 10, 10),
+                        pos);
```

Comment:
CI caught a test failure here:

```
  /github/workspace/dartsim/src/WorldFeatures_TEST.cc:202: Failure
  Eigen::Vector3d(10, 10, 10) and pos ([10 10 10] and 
[0.50050000000579775                   0  2.5005000000132527]) are not equal
```

Reply:
should `pos` be changed to `posNoGravity`?

Reply:
I think you are right.

Reply:
updated in [5f4c033](https://github.com/gazebosim/gz-physics/pull/633/commits/5f4c03345b07f5b29a3be134026fde0045dc4c63)